archives/archives.go

```diff
@@ -966,6 +968,19 @@ func ArchiveActiveOrgs(rt *runtime.Runtime) error {
 	timeTaken := dates.Now().Sub(start)
 	slog.Info("archiving of active orgs complete", "time_taken", timeTaken, "num_orgs", len(orgs))
 
+	rt.CW.Queue(types.MetricDatum{MetricName: aws.String("ArchiveElapsed"), Value: aws.Float64(timeTaken.Seconds()), Unit: types.StandardUnitSeconds})
```

Comment:
these are all sent once a night so we don't need to use the queue functionality .. see how it's done in indexer

Reply:
Okay